packages/types/src/clerk.ts

```diff
@@ -87,8 +87,12 @@ export interface Clerk {
   /** Clerk Flag for satellite apps. */
   isSatellite: boolean;
 
+  /** Clerk Instance type is defined from the Publishable key */
   instanceType?: InstanceType;
 
+  /** Clerk flag for loading Clerk in a standard browser setup */
+  isStandardBrowser?: boolean;
```

Comment:
❓ isn't this getter always available? Shouldn't we use `isStandardBrowser` instead of `isStandardBrowser?`?